### ISSUE TITLE
SDP (Software Development Plan) - drafting/reviewing it

### ISSUE THREAD

**Issue by dario-abbondanza** [tur-learning/CIS1051-python#26]
Just imlement my corrections.

**Comment by andreagalle** [comment]
I agree with you Dario, that’s a good starting point for his Final Project. 

@ben-vanzyll-temple push here on GitHub the revised SDP such that we can then move to the most exciting parts.

We are waiting for this to close this issue and grade the assignment. Then, have a look at the next issues/assignments.
